redi-connect-front/src/components/organisms/RediHero.scss

```diff
@@ -0,0 +1,37 @@
+@import "~bulma/sass/utilities/_all";
+@import '../../styles/variables.scss';
+
+.hero {
+  &__column {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: flex-start;
+
+    &--heading.title {
+      line-height: 4.5rem;
+      margin-bottom: 1rem;
+
+      @include mobile() {
+        font-size: 3rem;
+        line-height: 3.125rem;
+      }
+    }
+
+    &--content {
+      font-size: $size-4;
```

Comment:
Here the same comment from above... Please check out the props from the content component...